bfe_util/socket_util_windows.go

```diff
@@ -20,8 +21,8 @@ import (
 	"errors"
 )
 
-// GetsockoptMutiByte returns the value of the socket option opt for the
+// GetsockoptMultiByte returns the value of the socket option opt for the
 // socket associated with fd at the given socket level.
-func GetsockoptMutiByte(fd, level, opt int) ([]byte, error) {
-	return nil, errors.New("GetsockoptMutiByte is not supported on Windows OS.")
+func GetsockoptMultiByte(fd, level, opt int) ([]byte, error) {
+	return nil, errors.New("GetsockoptMultiByte is not supported on Windows OS.")
```

Comment:
Code Style
error strings should not be capitalized or end with punctuation or a newline

Reply:
I'm not sure line#27 what has to be changed, point it out please.

Run `staticcheck bfe_util/socket_util_windows.go`, no tips.

Reply:
`errors.New()` should be replaced with `fmt.Errorf()`, right?